src/transactions/utils.ts

```diff
@@ -1,12 +1,8 @@
 import { BigNumber } from '@waves/bignumber';
 import { Money } from '@waves/data-entities';
 import { binary, serializePrimitives } from '@waves/marshall';
-import {
-  base58Encode,
-  blake2b,
-  concat,
-  stringToBytes,
-} from '@waves/ts-lib-crypto';
+import { base58Encode, blake2b, concat, TBinaryIn } from '@waves/ts-lib-crypto';
+import { _fromRawIn } from '@waves/ts-lib-crypto/conversions/param';
```

Comment:
I don't think it's a good idea to use these internal imports. May be just write our own checks directly?